scripts/temp-create-data.js

```diff
@@ -253,7 +257,25 @@ const createMetacolony = async (singerOrWallet) => {
     if (metacolonyQuery?.errors) {
       console.log('METACOLONY COULD NOT BE CREATED.', metacolonyQuery.errors[0].message);
     } else {
-      console.log(`Creating metacolony { name: "meta", colonyAddress: "${utils.getAddress(metacolonyAddress)}", profile: { displayName: "Metacolony" }, nativeToken: "${utils.getAddress(metacolonyTokenAddress)}", version: "${metacolonyVersion.toString()}" }`);
+      console.log(`Creating metacolony { name: "meta", colonyAddress: "${utils.getAddress(metacolonyAddress)}", nativeToken: "${utils.getAddress(metacolonyTokenAddress)}", version: "${metacolonyVersion.toString()}" }`);
```

Comment:
Is the temp script necessary anymore ?

We should decide what to do with it. Either:
- remove it and create everything manually
- keep it, but update it so that it creates proper colonies (with extensions, token authorities and everything)

Reply:
Yeah, maintaining it is a bit of a pain. But it's nice to have the script that bootstraps everything for you.
One option could be to use some headless browser (cypress, puppeteer) to programmatically click through the user and then colony creation wizard, but I don't know how feasible that would be

Reply:
We had that before, and it more or less work, however it provided quite a bit of overhead and ate alot of resources.

I guess once we stabilize the core features, maintaining this won't be as much of a problem anymore